src/App.js

```diff
@@ -1,41 +1,64 @@
-import React, { useEffect, useState } from "react"
-import UserTable from "./tables/UserTable"
-import AddEmployeeForm from "./forms/AddEmployee"
-import EditUserForm from "./forms/EditUser"
-import "./App.css"
+import React, { useEffect, useState } from "react";
+import UserTable from "./tables/UserTable";
+import AddEmployeeForm from "./forms/AddEmployee";
+import EditEmployeeForm from "./forms/EditEmployee";
+import "./App.css";
 
-import { getEmployees, createEmployee, deleteEmployee } from './api/EmployeeService'
+import {
+  getEmployees,
+  createEmployee,
+  deleteEmployee,
+  updateEmployeeConst,
+} from "./api/EmployeeService";
 
 const App = () => {
-  const initialFormState = { id: null, name: "", username: "" }
-  const [currentUser, setCurrentUser] = useState(initialFormState)
+  const initialFormState = {
+    id: null,
+    firstName: "",
+    middle: "",
+    lastName: "",
+  };
+  //const [currentUser, setCurrentUser] = useState(initialFormState);
 
-  const [employees, setEmployees] = useState([])
-  const [editing, setEditing] = useState(false)
+  const [employees, setEmployees] = useState([]);
+  const [editing, setEditing] = useState(false);
+  const [employee, setEmployee] = useState(initialFormState);
 
   // Get employees
   useEffect(async () => {
-    const employees = await getEmployees()
-    setEmployees(employees)
-  }, [])
+    const employees = await getEmployees();
+    setEmployees(employees);
+  }, []);
 
-  const editRow = (user) => {
-    setEditing(true)
-    setCurrentUser(user)
-  }
+  const editRow = (employee) => {
+    console.log(employee);
+    setEditing(true);
+    setEmployee(employee);
+  };
 
   const addEmployee = (employee) => {
-    console.log({ employee })
-    employee.id = employees.length + 1
-    setEmployees([...employees, employee])
-    createEmployee({ body: employee })
-  }
+    console.log({ employee });
+    employee.id = employees.length + 1;
+    setEmployees([...employees, employee]);
+    createEmployee({ body: employee });
+  };
 
   const delEmployee = (id) => {
-    console.log({ id })
-    setEmployees(employees.filter((employee) => employee.id !== id))
-    deleteEmployee(id)
-  }
+    console.log({ id });
+    setEmployees(employees.filter((employee) => employee.id !== id));
+    deleteEmployee(id);
+  };
+
+  const updateEmployee = (id, updatedEmployee) => {
+    setEditing(false);
+    setEmployee(
+      employees.map((employee) =>
+        employee.id === id ? updatedEmployee : employee
+      )
+    );
+    updateEmployeeConst({ id: id, body: updatedEmployee });
+    console.log({ updatedEmployee });
+  };
```

Comment:
Remove `console.log({ editing });`

Also can you change `CRUD App with Hooks` to `CRUD App - Oracle Database`